cryptomator/index.md

```diff
@@ -96,7 +96,7 @@ When `Preferences → Cryptomator → Auto detect and open vault in browser` is
 ````{tabs}
 ```{group-tab} Cyberduck
 
-Choose the *Cryptomator* button in the toolbar or the option in the *File* menu to unlock and lock a Cryptomator Vault.
+Choose the *Cryptomator* button in the toolbar or *File → Unlock Vault* and *File →Lock Vault* menu to unlock or lock a vault respectively.
```

Comment:
I would change it to:
*File → Lock Vault* menu item to unlock or lock a Vault respectively.